drivers/wifi/infineon/Kconfig.cyw43xxx

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig WIFI_CYW43XXX
+	bool "Infineon CYW43xxx SoC Wi-Fi support"
+	select THREAD_CUSTOM_DATA
+	select USE_INFINEON_SDIO
+	select WIFI_OFFLOAD
+	select NET_L2_WIFI_MGMT
+	select WIFI_USE_NATIVE_NETWORKING
+	select ABSTRACTION_RTOS_COMPONENT_ZEPHYR
+	depends on DT_HAS_INFINEON_CYW43XXX_WIFI_SDIO_ENABLED
+	help
+	  Enable Infineon CYW43xxx SoC Wi-Fi support.
+	  Infineon's AIROC™ Wi-Fi & combos portfolio integrates IEEE 802.11a/b/g/n/ac/ax Wi-Fi and Bluetooth® 5.2
```

Comment:
This reads like a marketing blurb instead of an option description. Not sure what our guidelines are for this, but it seems out of place with other descriptions.

Reply:
updated.